### PR TITLE
Removed erroneous Wei units for intrinsic gas

### PR DESCRIPTION
**Pull request by AntoineRondelet**

### CODE REVIEW

**Review by dtebbs — APPROVED** [review]
LGTM

File: chapters/chap01-sec02.tex [hunk 1]
```diff
@@ -6,7 +6,7 @@ \section{Ethereum}\label{preliminaries:ethereum}
...
 
 To prevent unbounded state transitions in the state machine, each instruction executed by the \evm~is associated with a cost in \wei, referred to as ``the gas necessary to run the operation''. The ``gas cost'' of a transaction needs to be paid by the transaction originator (deduced from their account balance), and is awarded to the miner (added to their account balance) who successfully mines the block containing the transaction.
-In addition to the cost of every instruction executed as part of a state transition, every transaction has an intrinsic ``gas cost'' of $\txDefaultGas$ \wei~\cite[Appendix G]{ethyellowpaper}. Bounding modifications to the $\ethereum{}$ state by the amount of \wei~held in the transaction originator's account allows the system to avoid the Halting problem\footnote{\url{https://en.wikipedia.org/wiki/Halting\_problem}} and protects against a range of Denial of Service (\dos) attacks.
```
Comment (mpzajac) [inline chapters/chap01-sec02.tex:9]: Proposal: 
In addition to the cost of every instruction executed as part of a state transition, every transaction has an intrinsic cost of $\txDefaultGas$ gasi~\cite[Appendix G]{ethyellowpaper}.

File: notations.tex [hunk 2]
```diff
@@ -83,7 +83,7 @@
 \nomenclature[F]{$\jsin, \jsout, \jsmax$}{The number of inputs and outputs of a joinsplit and $\jsmax = \max\smallset{\jsin, \jsout}$}
 \nomenclature[F]{$\mkTreeDepth$}{The depth of the Merkle tree used to store commitments}
 \nomenclature[F]{$\addressLen$}{The bit-length of an \ethereum~address \nomunit{$160\, bits$}}
-\nomenclature[F]{$\txDefaultGas$}{The default/intrinsic gas cost of an \ethereum~transaction \nomunit{$21 000\, \wei$}}
+\nomenclature[F]{$\txDefaultGas$}{The default/intrinsic gas cost of an \ethereum~transaction \nomunit{$21 000$}}
```
Comment (mpzajac) [inline notations.tex]:
```suggestion
\nomenclature[F]{$\txDefaultGas$}{The default/intrinsic cost of an \ethereum~transaction \nomunit{$21 000$\, gas}}
```

**Review by mpzajac — COMMENTED** [review]
Proposed minor modificaitons -- instead of "gas cost of 21000" let's say "cost of 21 000 gas"

File: chapters/chap01-sec02.tex (hunk 1 above)
Comment (AntoineRondelet) [reply chapters/chap01-sec02.tex:9]: That works. I'll use this, thanks.

**Pull request merged**